packages/eventcatalog/pages/events/[name].tsx

```diff
@@ -46,6 +47,18 @@ export const getComponents = ({ event, schema, examples }: any) => ({
       </section>
     );
   },
+  SchemaViewer: ({ title = 'Event Schema' }: { title: string }) => {
+    if (!schema) return null;
+
+    return (
+      <section className="mt-8 xl:mt-10">
+        <h2 id="activity-title" className="text-lg font-medium text-gray-900 underline">
```

Comment:
Think we can remove the `id` here
```suggestion
        <h2 className="text-lg font-medium text-gray-900 underline">
```